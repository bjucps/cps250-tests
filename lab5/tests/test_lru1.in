ape
bat
cat
ape
bat
cat
dog
bat
emu
dog
cat
