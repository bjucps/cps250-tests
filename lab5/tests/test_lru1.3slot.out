ape: MISS (1/3 full; no eviction)
bat: MISS (2/3 full; no eviction)
cat: MISS (3/3 full; no eviction)
ape: HIT
bat: HIT
cat: HIT
dog: MISS (3/3 full; evicted 'ape')
bat: HIT
emu: MISS (3/3 full; evicted 'cat')
dog: HIT
cat: MISS (3/3 full; evicted 'bat')
--------------------------------
FINAL CACHE CONFIGURATION:
--------------------------------
cat
dog
emu
